.eslintrc.yaml

```diff
@@ -39,8 +74,4 @@ rules:
       filePathPattern: ".*\\.unit\\.test\\..*"
 parserOptions:
   sourceType: module
-overrides:
-  - files: "**/__tests__/**"
-    rules:
-      no-magic-numbers: off
-      "@typescript-eslint/no-unused-vars": off
+root: true
```

Comment:
nit-picky comment:
should we use `false` instead of `off` since we have properties with `boolean: true` in the same file?

No need to chage things though 😋

Reply:
@JillieBeanSim Yep, I have consolidated the rules into the `.eslintrc.yaml` at the root of the repo, but I noticed that without specifying the `tsconfig.json` path to ESlint (for each package that uses one), the ESlint VScode plugin would not parse the TypeScript files.

I will do some further experimentation to try to remove them entirely, but for now I have removed all of the rules from those `.yaml` files except the root - the remaining options are for the parser to be happy 😅

Reply:
oof, sorry I completely missed over that part 🤣 

From my understanding of the ESlint docs, `root` is a boolean (see [here](https://eslint.org/docs/latest/use/configure/configuration-files)):

> ESLint then searches up the directory structure, merging any .eslintrc files it finds along the way until reaching either an .eslintrc file with root: true or the root directory

We might be able to omit this entirely - going to try removing it and see what happens. But I agree that we should stick with one convention (`false` OR `off`) when applicable